### DISCUSSION TITLE
[Question] bridge. for mobile devices

### DISCUSSION THREAD

**Discussion by Yasserpro312** [bridge-core/editor#75] [Q&A] [answered]
Make a bridge app for mobile devices pls

**Comment by outercloudstudio** [comment]
You can visit the app on mobile using safari and it should still work.

**Comment by FrederoxDev** [comment, marked as answer]
If you are using bridge on iOS you can download it by pressing the share button ![image](https://user-images.githubusercontent.com/69014593/136044930-4c3fea50-4e2c-44fb-8f17-a256c6381f7b.jpeg) 
at the bottom of your screen and then pressing 
"Add to Home Screen"![image](https://user-images.githubusercontent.com/69014593/136045012-e6f0c832-57cd-4693-a28a-8c535d13cddb.jpeg)
